metrics_exporter.py

```
@@ -208,20 +210,32 @@ def main():
     Run metrics exporter.
     """
 
+    logger = logging.getLogger()
+    stop_event = threading.Event()
+
+    def _handle_shutdown(signum, frame):
```

Ruff (0.14.14)

[warning] Unused function argument: `frame`

(ARG001)